### PR TITLE
Add Jetty Alerts

### PR DESCRIPTION
**Pull request by Mrod1598**
Add jetty alerts based around the new metrics receiver as well as the jetty access log receiver.

### CODE REVIEW

File: alerts/jetty/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Alerts for Jetty in the Ops Agent
```
Comment (adeychman) [inline alerts/jetty/README.md:1]: Could we please add a description of what does this alert policy do and why is it important? 

Similar to this README https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/156/files#diff-7a1067bb99e3087c641d43fcc10cda016f5725e09ab8050c46fab4a9cd3e761d

Comment (adeychman) [reply alerts/jetty/README.md:1]: Oh, I see it down below on line 32. How about we move higher to first explain what this does, and then we can share the implementation details

File: alerts/jetty/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The filter for the metric should be: 
+```
+logName:"jetty_access"
+httpRequest.status > 299
+```
+
+## Failed Config Parsing
+
+There are logs at WARN level at start up that Identify failed config parsing for modules.
```
Comment (adeychman) [inline alerts/jetty/README.md:34]: Should `Identify` be lowercase or is this a field?

File: alerts/jetty/README.md (hunk 1 above)
Comment (Mrod1598) [reply alerts/jetty/README.md:1]: Updated

File: alerts/jetty/long_thread_queue.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "displayName": "Jetty - Long Thread Queue",
+  "documentation": {
+    "content": "If 'jetty.thread.queue.count' is above 10, which is the default total amount of threads, or a user defined value, it shows that your server can't keep up and improvements in performance should be looked for.",
```
[the remark this answers is not in the document]
Comment (Mrod1598) [reply alerts/jetty/long_thread_queue.json]: Updated

File: alerts/jetty/README.md (hunk 2 above)
Comment (Mrod1598) [reply alerts/jetty/README.md]: Whoops, updated.

File: alerts/jetty/access_log_failures.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        "aggregations": [
+          {
+            "alignmentPeriod": "60s",
+            "crossSeriesReducer": "REDUCE_SUM",
+            "perSeriesAligner": "ALIGN_SUM"
+          }
+        ],
+        "comparison": "COMPARISON_GT",
+        "duration": "0s",
+        "filter": "metric.type=\"logging.googleapis.com/user/jetty.request.failed.count\"",
```
Comment (chanseokoh) [inline alerts/jetty/access_log_failures.json:21]: resource.type must be specified.

File: alerts/jetty/failed_config_parsing.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "userLabels": {},
+  "conditions": [
+    {
+      "displayName": "Jetty - Failed Config Parsing",
+      "conditionMatchedLog": {
+        "filter": " (jsonPayload.message =~ \".*WARN  : Unknown Arguments detected.  Consider using --dry-run or --exec\" OR jsonPayload.message =~ \"Argument\\:.*\\(interpreted as a JVM argument\\, from \\<command-line\\>\\)\")"
+      }
+    }
+  ],
+  "alertStrategy": {
```
Comment (chanseokoh) [inline alerts/jetty/failed_config_parsing.json:16]: Error:
```
Field alertStrategy.notificationRateLimit had an invalid value: log-based alert policies need to specify a notification rate limit
```

Comment (Mrod1598) [reply alerts/jetty/failed_config_parsing.json:16]: Updated

File: alerts/jetty/access_log_failures.json (hunk 4 above)
Comment (Mrod1598) [reply alerts/jetty/access_log_failures.json]: Updated

**Review by chanseokoh — APPROVED** [review]

**Pull request merged**